qemu/tests/cfg/qemu_guest_agent.cfg

```diff
@@ -138,6 +138,8 @@
             image_snapshot = yes
             ppc64le:
                 image_snapshot = no
+            cmd_check_openssl = "which openssl"
+            cmd_install_openssl = "dnf -y install openssl"
```

Comment:
Hi @6-dehan I recommend you to use `utils_package.package_install` to install the package.
https://github.com/avocado-framework/avocado-vt/blob/c610f9b313e71a76d3343e624f3d9efd58115d84/virttest/utils_package.py#L227

Reply:
done. Please help to review when you're free.